runtime/doc/eval.txt

```diff
@@ -7175,6 +7177,15 @@ log10({expr})						*log10()*
 <
 		{only available when compiled with the |+float| feature}
 
+lshift({expr1}, {expr2})					*lshift()*
+		Bitwise left shift {expr1} by {expr2}. The arguments must be
+		Numbers.  Other types of arguments cause an error.
```

Comment:
Usually bit shifts in C are done with unsigned (although it's also possible with signed).
Vim script does not have unsigned.

We should document how to behaves with the sign bit.  In other words, what is the result of `lshift(-2, 1)`?
And we should test with negative numbers too.

In Java, left-shift has 2 different operators `>>` (sign extension) and `>>>` (shifts a zero into the leftmost position).    See: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/op3.html  So maybe we need an extra (optional?) parameter to indicate whether to extend sign or not?

Reply:
Thanks for the review. Yes. We can add an option to rshift() to deal with sign extension.